tests/vm_startup_test.go

```diff
@@ -88,7 +88,7 @@ var _ = Describe("Virtual Machines Startup", func() {
 			var nslookupOutput []byte
 			Eventually(func() error {
 				var nslookupErr error
-				nslookupOutput, nslookupErr = exec.Command("nslookup", fmt.Sprintf("-port=%s", dnsPort), fmt.Sprintf("%s.%s.%s.%s", interfaceName, vmiName, testNamespace, domain), dnsIP).CombinedOutput()
+				nslookupOutput, nslookupErr = exec.Command("nslookup", fmt.Sprintf("-port=%s", dnsPort), fmt.Sprintf("%s.%s.%s.%s.", interfaceName, vmiName, testNamespace, domain), dnsIP).CombinedOutput()
```

Comment:
Please consider using nosearch instead adding a dot

Reply:
We use dot for the same purpose in the zone file as well. I prefer to stick to this standard.